_vector-search/optimizing-storage/binary-quantization.md

```diff
@@ -117,6 +117,53 @@ PUT my-vector-index
 ```
 {% include copy-curl.html %}
 
+## Enhancing search quality with ADC and RR
+
+If the recall is lacking, you can specify asymmetric distance computation (ADC) or random rotation (RR) in the index mapping to apply enhanced search capabilities. These techniques are available in OpenSearch 3.2 and later versions.
+
+ADC maintains a full-precision query vector while rescaling it to have meaningful distance computations against binary-quantized document vectors. This asymmetric approach preserves more information about the query vector, boosting search quality without significant memory penalty. ADC is supported for 1-bit quantization only.
+
+Random rotation addresses the issue where binary quantization gives equal weight to each vector dimension during the quantization process. By rotating the distribution, RR can "smooth" variance (information) from high-variance dimensions into low-variance dimensions, preserving more information during the 32x compression process. RR is supported for 1-bit, 2-bit, and 4-bit quantization.
+
+For optimal performance and recall enhancement, use both ADC and RR together:
+
+```json
+PUT vector-index
+{
+  "settings" : {
+    "index": {
+      "knn": true
+    }
+  },
+  "mappings": {
+    "properties": {
+      "vector_field": {
+        "type": "knn_vector",
+        "dimension": 8,
+        "method": {
+            "name": "hnsw",
+            "engine": "faiss",
+            "space_type": "l2",
+            "parameters": {
+              "encoder": {
+                "name": "binary",
+                "parameters": {
+                  "bits": 1,
+                  "random_rotation": true,
+                  "enable_adc": true
+                }
+              }
+            }
+        }
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+**Note:** ADC and RR impact search or indexing performance, so they are opt-in features. ADC may introduce a moderate latency increase due to full-precision distance computations, while RR primarily affects indexing latency as vectors must be rotated during the process.
```

Comment:
```suggestion
ADC and RR impact search or indexing performance, so they are disabled by default. ADC may introduce a moderate latency increase because of full-precision distance computations, while RR primarily affects indexing latency because vectors must be rotated during the process.
{: .note}
```